src/it/scala/ApiServiceISpec.scala

```diff
@@ -146,6 +146,20 @@ class ApiServiceISpec
 
     }
 
+    "returns agent summaries message when given a valid ListAgentSummaryRequest" in { f =>
+      val textMessageConfig = generateValidMessageConfig()
+        .copy(messageTypeId = Some("OSLISAGT001"))
+        .copy(contents = s"""{
+                            |    "type" : ["CorporateBody","Person"],
+                            |    "authority-file" : false,
+                            |    "depository" : false,
+                            |    "version-timestamp" : ""
+                            |}""".stripMargin)
```

Comment:
I think this payload should just be
```
{
    "type" : ["CorporateBody","Person"]
}
```
We should also another test for
```
{
    "type" : ["CorporateBody"],
    "depository" : true
}
```

Reply:
Sorry, my bad. It should be `"Corporate Body"` not `"CorporateBody"` as per the schema.